### PR TITLE
[Task-49]-Change the menu and button elements

### PR DESCRIPTION
**Pull request by rajiravenantech**
Change the menu and button elements

### CODE REVIEW

File: src/app/components/Sidebar/Sidebar.tsx [hunk 1]
```diff
@@ -125,34 +127,27 @@ const Sidebar: React.FC = () => {
...
+        { label: "Wishlist", route: "/ecommerce/wishlist" },
       ],
     },
     { label: "Import/Export", route: "/import-export-element" },
     { label: "CRUD Component", route: "/crud/list" },
-    { label: "Subscription Plan", route: "/subscription-plan" }
+    { label: "Subscription Plan", route: "/subscription-plan" },
   ];
+  const [open, setOpen] = React.useState(true);
+  const [menuOpen, setMenuOpen] = React.useState(true);
```
Comment (esakkirajaravenantech) [inline src/app/components/Sidebar/Sidebar.tsx:138]: Remove usused states.

File: src/app/components/Sidebar/Sidebar.tsx [hunk 2]
```diff
@@ -125,34 +127,27 @@ const Sidebar: React.FC = () => {
...
-  const handleMouseLeave = () => {
-    if (open && !isMobile) {
-      handleDrawerClose();
-    }
-    setHover(false);
+  const handleDrawerToggle = () => {
+    setMenuOpen(!open);
   };
 
   const handleEcommerceClick = () => {
```
Comment (esakkirajaravenantech) [inline src/app/components/Sidebar/Sidebar.tsx:149]: Remove console.log

File: src/app/components/Sidebar/Sidebar.tsx [hunk 3]
```diff
@@ -205,17 +201,71 @@ const Sidebar: React.FC = () => {
               >
                 Admina
```
Comment (esakkirajaravenantech) [inline src/app/components/Sidebar/Sidebar.tsx:202]: Add some modification to the name like increase font size, change colour and font family.

File: src/app/globals.css [hunk 4]
```diff
@@ -28,15 +28,44 @@ header {
...
   background-color: transparent;
 }
 
+.sideMenuLink {
+  color: #000;
+}
+
+.activeMenu {
+  background-color: #4a4949;
+  color: #fff;
```
Comment (esakkirajaravenantech) [inline src/app/globals.css:48]: Change active menu background color

**Review by esakkirajaravenantech — CHANGES_REQUESTED** [review]

File: src/app/components/Sidebar/Sidebar.tsx (hunk 1 above)
Comment (rajiravenantech) [reply src/app/components/Sidebar/Sidebar.tsx]: Removed unused state.

File: src/app/components/Sidebar/Sidebar.tsx (hunk 2 above)
Comment (rajiravenantech) [reply src/app/components/Sidebar/Sidebar.tsx:149]: Removed console.log

File: src/app/components/Sidebar/Sidebar.tsx (hunk 3 above)
Comment (rajiravenantech) [reply src/app/components/Sidebar/Sidebar.tsx:201]: Change the logo color and font size

File: src/app/globals.css (hunk 4 above)
Comment (rajiravenantech) [reply src/app/globals.css:48]: Changed the active menu color

**Review by esakkirajaravenantech — APPROVED** [review]

**Pull request merged**